addons/web/static/src/core/utils/colors.js

```diff
@@ -198,6 +198,28 @@ export function convertCSSColorToRgba(cssColor) {
     // Note: however, if ever implemented be careful of 'white'/'black' which
     // actually are color names for our color system...
 
+    // Check if cssColor is a color() functional notation allowing colorspace
+    // with implicit sRGB.
+    // "<color()>" allows to define a color specification in a formalized
+    // manner. It starts with the "color(" keyword, specifies color space
+    // parameters, and optionally includes an alpha value for transparency.
+    const regexColor = "color\\(.+\\);?";
+    const regex = new RegExp(regexColor);
+    if (regex.test(cssColor)) {
+        const canvasEl = document.createElement("canvas");
+        canvasEl.height = 1;
+        canvasEl.width = 1;
+        const ctx = canvasEl.getContext("2d");
+        ctx.fillStyle = cssColor;
+        ctx.fillRect(0, 0, 1, 1);
+        const data = ctx.getImageData(0, 0, 1, 1).data;
+        return {
+            red: data[0],
+            green: data[1],
+            blue: data[2],
+            opacity: parseInt(data[3] / 2.55), // Convert 0-255 to percentage
```

Comment:
Should it really be parsed to int ? The doc says it returns a float

Reply:
My bad. Corrected.